src/Sentry.Unity.Editor.iOS/SentryXcodeProject.cs

```diff
@@ -69,9 +69,9 @@ public void AddSentryFramework()
 
             _project.AddFileToEmbedFrameworks(mainTargetGuid, frameworkGuid); // Embedding the framework because it's dynamic and needed at runtime
 
-            _project.SetBuildProperty(mainTargetGuid, "FRAMEWORK_SEARCH_PATHS", "$(inherited)");
+            _project.AddBuildProperty(mainTargetGuid, "FRAMEWORK_SEARCH_PATHS", "$(inherited)");
```

Comment:
Do we have any tests to validate this change?

Reply:
We do! Now!